### ISSUE TITLE
Some implementation queries on dropout, vgg16 last feature map tapping

### ISSUE THREAD

**Issue by KKNakka** [rohitgirdhar/AttentionalPoolingAction#23]
1. It seems like dropout layer before the topdown attention logits has missed the is_training argument (line 283 in nets_factory.py). Kindly check this

2. In vgg.py, the vgg16 function (line 166-170) uses the relued output of conv5_2 (not pre-relu conv5_3) as the final convolutional endpoint. And the relu activation of conv5_3 is missing from the architecture.  Since we are loading the imagenet weights, the architecture has to be consistent. Please share your thoughts.

**Comment by rohitgirdhar** [comment]
1. I don't think it's required because I define it in the `arg_scope` [here](https://github.com/rohitgirdhar/AttentionalPoolingAction/blob/248f710d7f3fc87cc04e0191dee249bfa34c001e/models/slim/nets/nets_factory.py#L143). Every call to `slim.dropout` within that block is passed in the `is_training` argument defined in the `arg_scope`.
2. I think you're right, I've fixed it now. Thanks for pointing it out. In any case it shouldn't affect the results since I don't use VGG networks in this work.
